README.md

```diff
@@ -31,6 +31,8 @@ This provider's versions are able to install and manage the following versions o
 |-|-|-|
 |OpenStack Provider v1alpha1 (ea309e7f)|✓|✓|
 
+Kubeadm and Kubelet versions are defined in `spec.versions.kubelet` of `cmd/clusterctl/examples/openstack/out/machines.yaml` which is generated by running the `generate-yaml.sh`.
```

Comment:
nit: maybe mention the template machines.yaml.template as well? and you may mention 
cmd/clusterctl/examples/openstack/out/machines.yaml is by default ,it depends where you run generate-yaml.sh